### ISSUE TITLE
[$35] Show loading indicator during sharing file with all members

### ISSUE THREAD

**Issue by maxceem** [appirio-tech/connect-app#2863]
When we upload a file and click **All project members** or **Share with selected members**, it hangs for a second because it's doing some backend job. See video demo https://monosnap.com/file/YHfEZxCvg23R3JCExvUXiNKyIXFcYo

![image](https://user-images.githubusercontent.com/146016/53411936-e3573680-3a02-11e9-8a42-1b027471a249.png)

We should show some the loading process is going on after we click **All project members** or **Share with selected members** until the popup is closed:
- make **All project members** and **Share with selected members** button disabled after pressing any of these buttons
- show loading indicator above the buttons. We already have a component for showing a loading indicator https://github.com/appirio-tech/connect-app/blob/dev/src/components/LoadingIndicator/LoadingIndicator.jsx which can be used here.

**Comment by r0hit-gupta** [comment]
I would like to work on this.

**Comment by maxceem** [comment]
@r0hit-gupta this is not yet open for pick up, as I'm waiting for some confirmation. 
Also, as per Bag Bush rules published in the [challenge specification](https://www.topcoder.com/challenges/30084798). One member can start working on another issue, only after raising a PR for the previous issue. As you working on this one https://github.com/appirio-tech/connect-app/issues/2833, please raise a PR for that issue, before picking up another one.

Note, as soon as this issue is open for pick up the one who first comment on it will be assigned to it.
Thanks.

**Comment by maxceem** [comment]
This issue is now open for pick up.

**Comment by sumitdaga** [comment]
please assign it to me! @maxceem

**Comment by rashmi73** [comment]
@maxceem please give it to me thank you

**Comment by maxceem** [comment]
@rashmi73 done

**Comment by sumitdaga** [comment]
@maxceem  i think i asked first! although the difference was only a few seconds :)

**Comment by maxceem** [comment]
@sumitdaga oh, sorry, I've missed your comment.

Yes, as you have asked first I'm assigning it to you.

Sorry @rashmi73, didn't see that @sumitdaga commented first. So I've reassigned to him.

**Comment by sumitdaga** [comment]
@maxceem  no worries :)  
> * show loading indicator above the buttons  
irrespective of which button was clicked show the loading indicator below the header right ?

**Comment by maxceem** [comment]
> irrespective of which button was clicked show the loading indicator below the header right ?

rigth

> and the modal can still be closed (x on top) right ?
yes, I think it's ok for now

**Comment by sumitdaga** [comment]
@maxceem  PR at #2867

**Comment by RishiRajSahu** [comment]
Working fine. Closing.
